radio/src/Makefile

```diff
@@ -52,7 +52,7 @@ DELAY = 5
 # PCB version
 # Values: 9X, 9X128, 9X2561, 9XR, 9XR128, 9XR2561, GRUVIN9X, MEGA2560, SKY9X, 9XRPRO, AR9X, TARANIS
 # Old values STD and STD128 are mapped to 9X and 9X128 for compatibility
-PCB = 9X
+PCB = 9XRPRO
```

Comment:
It will obviously always be a mess to keep mods in the makefile. Whch is again why it's not a good idea to do it... it also limits you to a single config.

I instead use multiple scripts for the various configs I use regularly in the form of 
```
make clean
make PCB=TARANIS PCBREV=REVPLUS HELI=YES GVARS=YES TRANSLATIONS=EN DEBUG=NO LUA=YES MIXERS_MONITOR=YES TIMERS=3 USB=JOYSTICK CLI=NO DANGEROUS_MODULE_FUNCTIONS=NO TARANIS_INTERNAL_PPM=NO

dfu-util-static -a 0 -d 0483:df11 --dfuse-address 0x08000000 -D opentx.bin
```
for example for the Taranis, that cleans, builds the version I want, then flashes it automatically.